test/token/ERC721/extensions/ERC721Consecutive.test.js

```diff
@@ -0,0 +1,128 @@
+const { constants, expectEvent, expectRevert } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+
+const ERC721ConsecutiveMock = artifacts.require('ERC721ConsecutiveMock');
+
+contract('ERC721Consecutive', function (accounts) {
+  const [ user1, user2, user3, receiver ] = accounts;
+
+  const name = 'Non Fungible Token';
+  const symbol = 'NFT';
+  const batches = [
+    { receiver: user1, amount: 0 },
+    { receiver: user1, amount: 3 },
+    { receiver: user2, amount: 5 },
+    { receiver: user3, amount: 0 },
+    { receiver: user1, amount: 7 },
+  ];
+
+  beforeEach(async function () {
+    this.token = await ERC721ConsecutiveMock.new(
+      name,
+      symbol,
+      batches.map(({ receiver }) => receiver),
+      batches.map(({ amount }) => amount),
+    );
+  });
+
+  describe('minting during construction', function () {
```

Comment:
We should test that it's not possible to mint in the constructor. Requires a different mock I think?